emails/views.py

```diff
@@ -1729,16 +1740,7 @@ def _handle_complaint(message_json: AWS_SNSMessageJSON) -> HttpResponse:
         profile.auto_block_spam = True
         profile.save()
 
-    disable_mask_on_complaint_flag, _ = get_waffle_flag_model().objects.get_or_create(
-        name="disable_mask_on_complaint",
-        defaults={
-            "note": (
-                "MPP-3119: When a Relay user marks an email as spam, disable the mask."
-            )
-        },
-    )
-    if disable_mask_on_complaint_flag.is_active_for_user(user):
-        _disable_masks_for_complaint(message_json)
+    _disable_masks_for_complaint(message_json)
```

Comment:
(suggestion, non-blocking) I recommend this be inside the for-loop where the the auto block spam is being set.

Reply:
OK, I misunderstood before when I suggested moving this inside the loop. The new code does this:

1. Loop through the `complained_recipients`, matching emails to Relay user's real email. Set the `auto_block_spam` flag for those users
2. Loop through the `destination` field in the mail message, looking for masks. Set those masks to blocking, and send an email
3. Log the actions taken for `complained_recipients`
4. Return an HTTP response like it is 2021

I think the problem is adding part 2 between parts 1 and 3. Instead, you should move the `complained_recipients` code to a new method, and have `_handle_complaint` call both of them. Unless, you want to ensure the masks belong to the complaining users, in which case it should move inside the loop.